src/backend/distributed/sql/udfs/invalidate_inactive_shared_connections/latest.sql

```diff
@@ -0,0 +1,10 @@
+CREATE OR REPLACE FUNCTION pg_catalog.invalidate_inactive_shared_connections()
```

Comment:
I moved this from test to UDF since we will use this in MX case.

Reply:
Thanks @SaitTalhaNisanci, @marcocitus noted that we need to invalidate the inactive shared connections on the MX worker nodes. And, we need a SQL interface anyway, so I can use this.

Reply:
Can we cherry pick this commit onto the relevant branch? It's hard for me to assess whether this is useful / necessary / right in the context of a different PR.

Reply:
Sorry I saw this message just now.